db/sql/400_api.sql

```diff
@@ -91,4 +93,16 @@ RETURNS setof json as $$
   SELECT cast(ST_AsGeoJSON(sc.*) AS json)
   FROM selected_cols AS sc;
 $$ LANGUAGE SQL VOLATILE;
+COMMENT ON FUNCTION api.get_observations_with_null_stop_id_4326 IS
+'Returns observations with null stop_id as GeoJSON features';
```

Comment:
Suggested: `'Returns observations with no stop_id value, but located within search_distance_m from the given stop_id, as GeoJSON features';`